components/FormationCard.tsx

```diff
@@ -1,10 +1,4 @@
-"use client";
```

Comment:
Wait does this default to server or client?

Reply:
for now this will stay as a client component until i refactor the rendering some more